.gitignore

```diff
@@ -14,3 +14,4 @@ coverage
 test/typescript/.idea/*
 test/typescript/*.js
 test/typescript/*.map
+package-lock.json
```

Comment:
Because it's still a very experimmental technology and there have been a lot of bugs. Plus, it does not make the life of a maintainer easier. So, no, we should not use them.